src/vmm/src/device_manager/persist.rs

```diff
@@ -392,7 +390,8 @@ mod tests {
         let version_map = VersionMap::new();
         // These need to survive so the restored blocks find them.
         let _block_files;
-        let _tmp_sock_file;
+        let mut _tmp_sock_file = TempFile::new().unwrap();
```

Comment:
nit: doesn't need the underscore anymore since it's used.

```suggestion
        let mut tmp_sock_file = TempFile::new().unwrap();
```

Reply:
Thanks for the comment. I removed the leading underbar.